docs/groovy-script/mods/selectionguicrafting/sgc_category.md

```diff
@@ -0,0 +1,177 @@
+
+:::::::::: details mods.selectionguicrafting.sgc_category.newCategory() {open id="abstract"}
+- `String`. The Category ID, must be unique. The resource must be loaded via external methods. Requires not null.
+
+    ```groovy:no-line-numbers
+    id(String)
+    ```
+
+- `ResourceLocation`. The path to the frame texture. The resource must be loaded via external methods. Can be overridden by the recipe.
+
+    ```groovy:no-line-numbers
+    frame(String)
+    ```
+
+- `ResourceLocation`. The path to the border texture. The resource must be loaded via external methods.
+
+    ```groovy:no-line-numbers
+    border(String)
+    ```
+
+- `ArrayList<GsSound>`. The sounds that will be played when the recipe is crafted. Can be overridden by the recipe.
+
+    ```groovy:no-line-numbers
+    sound(String, float, float)
+    ```
+
+- `ArrayList<GsParticle>`. The particles that will be spawned when the recipe is crafted. Can be overridden by the recipe.
+
+    ```groovy:no-line-numbers
+    particle(String, int, float)
+    ```
+
+- `GsEnum.QueueType`. If the recipes in this category can be queued. Can be overridden by the recipe. Allowed values are: 'true' or 'false'. DEFAULT is 'true'.
+
+    ```groovy:no-line-numbers
+    queueType(String)
+    queueType(boolean)
+    ```
+
+- `GsEnum.SoundType`. How the sounds will be played. Can be overridden by the recipe. Allowed values are: 'RANDOM' or 'COMBINED'. DEFAULT is 'RANDOM'.
+
+    ```groovy:no-line-numbers
+    soundType(String)
+    ```
+
+- `ResourceLocation`. The path to the background texture. The resource must be loaded via external methods.
+
+    ```groovy:no-line-numbers
+    background(String)
+    ```
+
+- `ResourceLocation`. The path to the decoration texture. The resource must be loaded via external methods.
+
+    ```groovy:no-line-numbers
+    decoration(String)
+    ```
+
+- `GsEnum.OutputType`. How the output will be handed to the player. Can be overridden by the recipe. Allowed values are: 'DROP' or 'INVENTORY'. DEFAULT is 'DROP'.
+
+    ```groovy:no-line-numbers
+    outputType(String)
+    ```
+
+- `String`. The display name of the category that will be shown in the GUI. Requires not null.
+
+    ```groovy:no-line-numbers
+    displayName(String)
+    ```
+
+- `ResourceLocation`. The path to the progress bar texture. The resource must be loaded via external methods. Can be overridden by the recipe.
+
+    ```groovy:no-line-numbers
+    bar(String)
+    ```
+
+- `GsEnum.BackgroundType`. How the background will be rendered. The resource must be loaded via external methods. Allowed values are: 'SINGLE_STRETCH' , 'TILE' or 'SINGLE_CUT'. DEFAULT is 'TILE'. 'TILE' uses a 16x16 texture and repeats it. 'SINGLE_STRETCH' stretches the texture ratio to fill the screen. 'SINGLE_CUT' stretches the texture while keeping the aspect ratio and cuts of anything that goes outside the gui. The texture is centered on the x and y axis.
```

Comment:
the `backgroundType` property should contain the default, not be inserted into the middle of this.
this fix is to highlight the weird whitespace and missing comma:
```suggestion
- `GsEnum.BackgroundType`. How the background will be rendered. The resource must be loaded via external methods. Allowed values are: 'SINGLE_STRETCH', 'TILE', or 'SINGLE_CUT'. DEFAULT is 'TILE'. 'TILE' uses a 16x16 texture and repeats it. 'SINGLE_STRETCH' stretches the texture ratio to fill the screen. 'SINGLE_CUT' stretches the texture while keeping the aspect ratio and cuts of anything that goes outside the gui. The texture is centered on the x and y axis.
```